src/utils.py

```diff
@@ -27,7 +31,7 @@ def tfidf_lr_model():
     return tfidf_model
 
 
-def bert_finetune_model():
+def bert_finetune_model()->Trainer:
     """loading the bert finetuned model"""
     bert_model = AutoModelForSequenceClassification.from_pretrained(
         "./models/bert_finetuned_model2"
```

Comment:
This test fails since this folder is not included in the repo.

Reply:
Do I have to remove this test ? The model is not there in the repo because it is huge file. As you mentioned before, I can keep all the large files in my drive and give the link